### PR TITLE
feat(server): bundle basemaps-server cli so its easier to install

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: packages/config/src/json/json.config.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ export class ConfigJson {
     for await (const tsPath of fsa.list(fsa.join(basePath, 'tileset'))) {
       if (!tsPath.endsWith('.json')) continue;
       const ts = await cfg.tileSet(await fsa.readJson(tsPath));
+      // console.log('LoadTileSet', )
```
Comment (Wentao-Kuang) [inline packages/config/src/json/json.config.ts:54]: Comment out code.

Comment (blacha) [reply packages/config/src/json/json.config.ts:54]: good point I was going to add logs to this.

File: packages/shared/src/cli/base.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      LogConfig.get().level = 'info';
+    }
+
+    const logger = LogConfig.get().child({ id: CliId });
+    logger.info(CliInfo, 'CliStart');
+    LogConfig.set(logger);
+
+    return super.onExecute();
+  }
+  protected onDefineParameters(): void {
```
Comment (Wentao-Kuang) [inline packages/shared/src/cli/base.ts:46]: What is this function for?

Comment (blacha) [reply packages/shared/src/cli/base.ts:46]: its a required function (I think?) this file was just copy pasted from @basemaps/cli to a central location

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**